docker-compose.yml

```
@@ -4,6 +4,7 @@ services:
     image: content-build:${IMAGE_TAG:-latest}
     build:
       context: .
+      target: base
     volumes:
       - .:/application
```

I think with the new Dockerfile changes this needs to be updated
```suggestion
      - .:/application/content-build
```